src/main/webapp/register.html

```diff
@@ -160,6 +160,13 @@ <h5 class="text-center lead">Fill in the following fields below.</h5>
                 value="" name="linkedin">
           </div>
         </div>
+        <div class="form-check">
+          <input class="form-check-input" type="checkbox" id="ok-shadow-check">
+          <label class="form-check-label" for="ok-shadow-check">
+            I am OK if another person shadows the interviews I have.
```

Comment:
Weird that this is worded differently than on the profile page.

Reply:
Good catch